gadgets/traceloop/README.mdx

```diff
@@ -32,14 +34,63 @@ Running the gadget:
 
 ## Guide
 
-First, we need to run an application that generates some events.
+System calls are the foundation of how applications interact with the operating system. When debugging production issues, understanding the exact sequence of system calls before a crash or failure can be invaluable. The traceloop gadget continuously captures these system calls, allowing you to "rewind" and see exactly what your application was doing when something went wrong.
+
+:::info
+Think of traceloop as a flight recorder for your applications - it's always recording, and when something goes wrong, you can review the exact sequence of events that led to the issue.
+:::
+
+### Flow of System Call Recording in Kubernetes
+
+Before diving into scenarios, it's important to understand how traceloop captures system calls across different components:
+
+**Application Pods**: The primary targets that generate system calls through normal operation.
+
+**eBPF Probes**: Kernel-level probes that capture system calls in real-time.
+
+**Traceloop Recorder**: The component that buffers and formats system call data.
+
+**Output Stream**: Real-time display of captured system calls with context.
+
+The overall flow of system call capture is as follows:
+
+```mermaid
+sequenceDiagram
+    participant App as Application Pods
+    participant Kernel as System Calls
+    participant eBPF as eBPF Probes
+    participant Traceloop as Traceloop Recorder
+    participant Output as Output Stream
+
+    App ->> Kernel: Generate syscalls
+    Kernel ->> eBPF: Kernel level interception
+    eBPF ->> Traceloop: Capture events
+    Traceloop ->> Output: Buffer & format data
+    Output ->> App: Display results
```

Comment:
I think we need another participant in this diagram. The Output of the `traceloop` gadget doesn't get directed to the app that was generating syscall events and then crashed.

It is sent to the user of the `traceloop` gadget itself